packages/xo-web/src/common/sorted-table/index.js

```diff
@@ -603,6 +603,14 @@ export default class SortedTable extends Component {
   }
   _setPage = this._setPage.bind(this)
 
+  goTo(id) {
+    const page = Math.ceil(
+      this._getItems().findIndex(item => item.id === id) /
+        this.props.itemsPerPage
+    )
+    this._setPage(page === 0 ? 1 : page)
```

Comment:
With 10 items per page, the item at position `10` (so the 11th item) is on page 2 but you'll set the page to `1`.
```js
this._setPage(Math.floor(
  this._getItems().findIndex(item => item.id === id) /
    this.props.itemsPerPage
) + 1)
```